DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/compute_units/src/compute_units.hpp

```diff
@@ -0,0 +1,37 @@
+//==============================================================
+// Copyright Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+#include <CL/sycl.hpp>
+#include <utility>
+
+namespace {
+template <class Func, template <std::size_t> class Name, std::size_t index>
```

Comment:
In modern C++, this is
```c++ 
template <typename Func, template <std::size_t> typename Name, std::size_t index>
```

Reply:
Thank you, I will fix this and remember it going forward.